src/olmo_core/config.py

```diff
@@ -271,11 +272,17 @@ def clean_data(d: Any, prefix: str) -> Any:
             raise OLMoConfigurationError(str(e))
 
     @classmethod
-    def from_file(cls: Type[C], path: PathOrStr, overrides: Optional[List[str]] = None) -> C:
```

Comment:
as far as I can tell, from_file is never used

Reply:
Would be nice to keep this, but have it call out to `.from_json` or `.from_yaml` depending on the filetype.